### ISSUE TITLE
[Mopidy]Home Assistant and remote MPD player not sound

### ISSUE THREAD

**Issue by yadenisenko** [Poeschl/Hassio-Addons#111]
Which addon?
Mopidy

Hello,
My configuration:
VPS on Debian 10
Home Assistant 0.116.4
Docker 19.03.13
Installed addon Mopidy and create media player:

media_player:
  - platform: mpd
    host: IP_my_server

I install MPD client for Android(Mpdroid, M.A.L.P) and set IP my VPS in settings and try run service from example:

entity_id: media_player.mpd
media_content_id: 'https://mydomain.tld/local/1.mp3'
media_content_type: music

MPD client get my request and i see url and filename, but when he running, then my android device do not playing sound
As if this file is empty, but in reality it is not.
The volume in android is at maximum and I have tried several android devices, the problem is the same everywhere.
Can you help me?
Mopidy conf:

local_scan: true
options:
  - name: http/enabled
    value: 'true'
  - name: stream/enabled
    value: 'true'
  - name: stream/protocols
    value: http
  - name: stream/protocols
    value: https
  - name: local/media_dir
    value: /share/mopidy/media

Full log Mopidy:
INFO     Starting Mopidy 2.2.2
INFO     Loading config from builtin defaults
INFO     Loading config from /var/lib/mopidy/.config/mopidy/mopidy.conf
INFO     Loading config from command line options
INFO     Enabled extensions: iris, mpd, http, file, stream, m3u, softwaremixer, moped, local-images, local, local-sqlite
INFO     Disabled extensions: spotify, gmusic
WARNING  Found gmusic configuration errors, the extension has been automatically disabled:
WARNING    gmusic/username must be set.
WARNING    gmusic/password must be set.
WARNING  Found spotify configuration errors, the extension has been automatically disabled:
WARNING    spotify/username must be set.
WARNING    spotify/client_secret must be set.
WARNING    spotify/password must be set.
WARNING    spotify/client_id must be set.
WARNING  Please fix the extension configuration errors or disable the extensions to silence these messages.
INFO     Starting Mopidy mixer: SoftwareMixer
INFO     Starting Mopidy audio
INFO     Starting Mopidy backends: StreamBackend, M3UBackend, FileBackend, LocalBackend
WARNING  Local media dir /share/mopidy/media does not exist or we lack permissions to the directory or one of its parents
INFO     Loaded 0 local tracks using sqlite
INFO     Starting Mopidy core
INFO     Audio output set to "autoaudiosink"
INFO     Starting Mopidy frontends: IrisFrontend, MpdFrontend, HttpFrontend
INFO     MPD server running at [::ffff:0.0.0.0]:6600
INFO     Starting Iris 3.33.0
INFO     Starting GLib mainloop
INFO     HTTP server running at [::ffff:0.0.0.0]:6680
INFO     New MPD connection from [::ffff:172.30.33.2]:6600

Simply put, I see the covers on the files, I see that the file is playing and time goes by. But there is no sound...
![photo_2020-10-22_15-08-24](https://user-images.githubusercontent.com/15608867/96895730-601b0c00-1495-11eb-8cae-5f194d8acf4e.jpg)

**Comment by Poeschl** [comment]
The mopidy addon here is depreciated, so I'm not changing it anymore. I guess it needs to change its audio output to Pulseaudio.
If you just want to play audio on your machine take a look at the `mpd` addon, which is a pure mpd server.

**Comment by yadenisenko** [comment]
Thanks, i tried using addon MPD without mopidy, but this has the same problem(The only thing that I saw suspicious in the logs was only: "exception: OutputThread could not get realtime scheduling, continuing anyway: sched_setscheduler failed: Operation not permitted")
I want to listen to sound from the MPD server on my phone with the MPD client. 
Sorry for asking, can you tell me where I can find a manual on how to change Pulseaudio output?

bash-5.0# ha audio info
audio:
  application:
  - addon: 8b6f0bc66c81d2b9ef76a3fc9b52b452
    index: 259
    mute: false
    name: Mopidy
    stream_index: 0
    stream_type: output
    volume: 1
  card: []
  input: []
  output:
  - applications:
    - addon: 8b6f0bc66c81d2b9ef76a3fc9b52b452
      index: 259
      mute: false
      name: Mopidy
      stream_index: 0
      stream_type: output
      volume: 1
    card: null
    default: true
    description: Dummy Output
    index: 0
    mute: false
    name: auto_null
    volume: 1
host: 172.30.32.4
version: "17"
version_latest: "17"

bash-5.0# pactl list sinks
Sink #0
        State: IDLE
        Name: auto_null
        Description: Dummy Output
        Driver: module-null-sink.c
        Sample Specification: s16le 2ch 44100Hz
        Channel Map: front-left,front-right
        Owner Module: 12
        Mute: no
        Volume: front-left: 65536 / 100% / 0.00 dB,   front-right: 65536 / 100% / 0.00 dB
                balance 0.00
        Base Volume: 65536 / 100% / 0.00 dB
        Monitor Source: auto_null.monitor
        Latency: 658553 usec, configured 2000000 usec
        Flags: DECIBEL_VOLUME LATENCY
        Properties:
                device.description = "Dummy Output"
                device.class = "abstract"
                device.icon_name = "audio-card"
        Formats:
                pcm

**Comment by Poeschl** [comment]
Please try the newest version of mpd (version 1.4.0). Its using the latest audio output.

The change to Pulseaudio must be done inside the addon. Also mopidy is doing some strange things inside a addon, therefore I stopped working on it. 

Hm, you might also look for a guide to how to stream the sound to your phone. Since mopidy and mpd are intended to play sounds over the audio interface of the host.

**Comment by yadenisenko** [comment]
Thanks! I will think about how to solve the problem.
